api/types/types.go

```diff
@@ -76,14 +84,17 @@ type Image struct {
 	Labels      map[string]string
 }
 
+// GraphDriverData returns Image's graph driver config info
+// when calling inspect command
 type GraphDriverData struct {
 	Name string
 	Data map[string]string
 }
 
+// ImageInspect contains response of Remote API:
 // GET "/images/{name:.*}/json"
 type ImageInspect struct {
-	Id              string
+	ID              string
```

Comment:
@vdemeester Do you mean that only `ID` field should be modified like this, or the `Parent`,'Comment', 'Created' etc fields below should also do these works? 
Same as below :p